frontend/src/components/StatusTags.vue

```diff
@@ -35,11 +46,55 @@ export default {
   },
   mixins: [shootItem],
   computed: {
-    filteredConditions () {
+    ...mapState([
+      'conditionCache'
```

Comment:
It is very strange to use the store as cache. Of course, I know not this PR. We should only keep known conditions in the store and create defaults on the fly. Since the getter already does caching.

Reply:
we were both right - there are hard-coded conditions in the store but they can be extended by `known conditions` using our dashboard configuration. Therefore I suggest to leave it for now as it requires more effort and is not related to this PR.